demo/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import HomeScreen from './src/screens/HomeScreen';
+import EditorScreen from './src/screens/EditorScreen';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
```

Comment:
are we sure it's ok to use react-native-navigation's competitor?

Reply:
RNN is not compliant with Expo since it has native code. [ui-lib](https://github.com/wix/react-native-ui-lib/blob/master/expoDemo/package.json) also used react-navigation for their expo app so I guess its OK.